tsconfig-node.json

```diff
@@ -17,13 +17,6 @@
     ]
   },
   "include": [
-    "./lib/Core/**/*",
-    "./lib/Models/**/*",
-    "./lib/Map/**/*",
-    "./lib/ModelMixins/**/*",
-    "./lib/Traits/**/*",
-    "./lib/Language/**/*",
-    "./lib/ReactViews/**/*",
-    "./lib/ThirdParty/**/*"
+    "./lib/**/*"
```

Comment:
Why does tsconfig-node.json override the include from tsconfig.json and leave out the tests and buildprocess files?